hack/generator/pkg/astmodel/comments.go

```diff
@@ -0,0 +1,102 @@
+package astmodel
+
+import (
+	"go/ast"
+	"strings"
+)
+
+// Utility methods for adding comments
+
+func addDocComment(commentList *[]*ast.Comment, comment string, width int) {
+	for _, c := range formatDocComment(comment, width) {
+		line := strings.TrimSpace(c)
+		if !strings.HasPrefix(line, "//") {
+			line = "//"+line
+		}
+
+		if *commentList==nil {
+			line = "\n"+line
+		}
+
+		*commentList = append(*commentList, &ast.Comment{
+			Text: line,
+		})
+	}
+}
+
+// formatDocComment splits the supplied comment string up ready for use as a documentation comment
+func formatDocComment(comment string, width int) []string {
+	// Remove markdown bolding,
+	text := strings.ReplaceAll(comment, "**", "")
+
+	// Turn <br> and <br/> into \n
+	text = strings.ReplaceAll(text, "<br/>", "\n")
```

Comment:
suggest living in sin and using `regex.ReplaceAllString` with `<br[^/>]*/?>`

Reply:
Nice

Reply:
Done.